reporting/terraform/main.tf

```diff
@@ -30,6 +31,26 @@ module "lambda_miro_inventory_transformer" {
     ES_DOC_TYPE = "miro_inventory_record"
   }
 
+  vhs_read_policy       = "${local.miro_inventory_vhs_read_policy}"
   trigger_topic_arn     = "${local.miro_inventory_topic_arn}"
   error_alarm_topic_arn = "${local.lambda_error_alarm_arn}"
 }
+
+module "lambda_sierra_transformer" {
+  source = "./reporting_lambda"
+
+  name        = "sierra_transformer"
```

Comment:
Minor: I'd like us to be better about namespacing our applications. We already have a “sierra_transformer” in the catalogue pipeline – maybe all these Lambdas could have their name prefixed with “reporting-”, or something?

(I checked the module, but couldn’t see anything doing this.)

Reply:
👍yes we also talked about this